expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java

```diff
@@ -76,6 +82,35 @@ private List<InputStream> getInputStreams(JsonNode indexDocument) {
         return inputStreams;
     }
 
+    private void injectScientificIndexStatus(JsonNode indexDocument) {
+        ((ObjectNode) indexDocument).set(ScientificIndex.SCIENTIFIC_INDEX_FIELD, fetchNviStatus(indexDocument));
+    }
+
+    private JsonNode fetchNviStatus(JsonNode indexDocument) {
+        return Optional.ofNullable(uriRetriever.fetchResponse(fetchNviCandidateUri(indexDocument), "application/json"))
+                   .filter(Optional::isPresent)
+                   .map(Optional::get)
+                   .filter(response -> response.statusCode() == 200)
```

Comment:
Bear in mind that you're swallowing all errors…we might want to catch this error to retry

Reply:
Throwing custom exception when failing:

```
    @Test
    void shouldThrowExpansionExceptionWhenUnexpectedErrorFetchingNviCandidateForPublication()
        throws ApiGatewayException {
        var publication = TicketTestUtils.createPersistedPublication(PUBLISHED, resourceService);
        var resourceUpdate = Resource.fromPublication(publication);
        var expansionService = expansionServiceReturningNviCandidate(publication, randomString(), 200);

        assertThrows(ExpansionException.class, () -> expansionService.expandEntry(resourceUpdate));
    }
```